backend/src/api/serializers.py

```diff
@@ -12,3 +15,35 @@ class NewsSerializer(serializers.ModelSerializer):
     class Meta:
         model = News
         fields = ('id', 'title', 'summary', 'source', 'href', 'tags', 'date')
+
+class RegisterSerializer(serializers.ModelSerializer):
```

Comment:
We can create a separate django app for auth stuff, to keep the different responsibilities of each section separated